### DISCUSSION TITLE
v3.2.0

### DISCUSSION THREAD

**Discussion by sojan-official** [chatwoot/chatwoot#8135] [Releases & Announcements]
## Changelog

- Ability to reply to a specific message in the agent dashboard
- Ability to send attachments while creating outbound conversations
- Delivery status updates for Twilio and live chat channels
- Support for link unfurling in Slack integration
- Dark theme support for knowledge base portals
- Ability to search contacts by company name
- Knowledge management in super admin for robin AI
- Support for multiple values in automation message content rule
- Support for assigning to self in macros
- Improved search across super admin for users and accounts
- Ability to view migration status in super admin
- Numerous performance improvements and bug fixes

###
Thanks to @smtbos, @LiamAshdown, @jordan-brough, @jsobralgitpush, @gitstart, @AqidaHaidari, @mazenkhalil, @BikashSah999,    for the contributions

<hr /><em>This discussion was created from the release <a href='https://github.com/chatwoot/chatwoot/releases/tag/v3.2.0'>v3.2.0</a>.</em>

**Comment by jtayrone** [comment]
Gratitude for the dedicated effort. As always, elevating the tool to another level. 👍😎👋🏻👋🏻
